build.gradle

```diff
@@ -344,6 +344,17 @@ project ('spring-pulsar-boot-autoconfigure') {
 	}
 }
 
+project ('spring-pulsar-docs') {
+	description = 'Spring Pulsar Code Snippets For Docs'
+
+	dependencies {
+		api "org.springframework.boot:spring-boot-starter:$springBootVersion"
```

Comment:
I wonder if we should instead use the spring-pulsar starter/auto-configure here to future proof additions from there?